src/RunTests/build/Microsoft.Build.RunVSTest.targets

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <UsingTask TaskName="Microsoft.Build.RunVSTestTask" AssemblyFile="$(MSBuildThisFileDirectory)netstandard2.0\Microsoft.Build.RunVSTest.dll" />
+  <Target Name="RunVSTest" AfterTargets="VSTest" Condition="$(IsTestProject)==true">
```

Comment:
the after here should be `MSBuildRunTests` or?

Reply:
No, the empty target was renamed to be VSTest, because that is the closest to Test as we can get probably. We should test that this won't break the existing vstest target though.

Reply:
I've updated the target name in the msbuild pr to reflect the new "VSTest" name

> the after here should be `MSBuildRunTests` or?

Reply:
Updated name to "Test", now that the target is opt in